bundles/org.openhab.binding.boschindego/src/main/java/org/openhab/binding/boschindego/internal/IndegoController.java

```diff
@@ -59,6 +60,7 @@ public class IndegoController {
 
     private static final String BASE_URL = "https://api.indego-cloud.iot.bosch-si.com/api/v1/";
     private static final String CONTENT_TYPE_HEADER = "application/json";
+    private static final int REQUEST_TIMEOUT_MS = 10000;
```

Comment:
How was this timeout value decided? Please keep in mind that this is a cloud service that might be slow from time to time. I have know idea what would be the best value, so for now just asking the question since the same value is used across all bindings.

Reply:
In some cases the mower is queued over cellular network, and it has to be woken up when asleep. These calls are pretty slow, so I'm pretty sure this timeout must be much higher (or dropped).

Reply:
Yes, i agree. 10 seconds may be to less. The bigger flaw from my point of view is, that's not configurable, but hard coded. Maybe 10 seconds are enough for most requests. But i would not assume this is true for every system and every server. This should be at least configurable using a system property or similar (which makes this change more complex).

Reply:
There was no scientific research to determine the general timeout for this PR. To me in general 10 second seemed reasonable, but i agree there might be some specific bindings http calls that may act slower.
 (even though i have never seen anything that slow)

Anyway, i'm happy to adjust any binding if needed. I tripled this one to 30 seconds.